pyzfp.pyx

```diff
@@ -170,6 +170,34 @@ cdef zfp_field* init_field(np.ndarray indata):
         raise ValueError("Invalid number of dimensions (valid: 1-4)")
 
 def compress(indata, tolerance=None, precision=None, rate=None, parallel=True):
+    """
+    Compress a numpy array using zfp.
+
+    Parameters
+    ----------
+    indata : numpy.ndarray
+        The data to compress.
+    tolerance : float, optional
+        The tolerance for the compressed data.
+        This will use ZFP in fixed-tolerance mode.
+        One of tolerance, precision, or rate must be specified.
+    precision : int, optional
+        The precision of the compressed data.
+        This will use ZFP in fixed-precision mode.
+        One of tolerance, precision, or rate must be specified.
+    rate : float, optional
+        The rate of the compressed data.
+        This will use ZFP in fixed-rate mode.
+        One of tolerance, precision, or rate must be specified.
+    parallel : bool, optional
```

Comment:
Should note the default is `True`. I've found that Cython doesn't let you see the function sig so the documentation has to be manually added (or you can add a pure python wrapper around the cython call).

Reply:
Did a quick test with `help(pyzfp.compress)` and it seemed to work fine.